proto/pocket/supplier/query.proto

```diff
@@ -52,7 +63,24 @@ message QueryAllSupplierRequest {
 }
 
 message QueryAllSupplierResponse {
-  repeated pocket.shared.Supplier                               supplier   = 1 [(gogoproto.nullable) = false];
+  repeated pocket.shared.Supplier                 supplier   = 1 [(gogoproto.nullable) = false];
+           cosmos.base.query.v1beta1.PageResponse pagination = 2;
```

Comment:
Is the addition of the `pagination` field on `QueryAllSupplierResponse` related to this PR?